lpa-submissions-e2e-tests/cypress/integration/planningOfficersReport.feature

```diff
@@ -0,0 +1,127 @@
+    When invalid files '<invalid_file_size>' have been selected
+    Then progress is halted with a message the file '<invalid_file_size>' 'is too big'
+    Examples:
+      | invalid_file_size      |
+      | upload_file_large.tiff |
+
+  Scenario Outline: AC7 LPA Planning Officer selects Invalid file format
+    Given Upload the Planning Officer’s report question is requested
+    When invalid files '<invalid_format>' have been selected
+    Then progress is halted with a message the file '<invalid_format>' 'format is incorrect'
+    Examples:
+      | invalid_format                     |
+      | upload-file-invalid-wrong-type.csv |
+
+  Scenario: AC8 LPA Planning Officer selects to return to previous page
+    Given Upload the Planning Officer’s report question is requested
+    And a file has been uploaded
+    When Back is then requested
+    Then the LPA Planning Officer is taken to the Task List
+    And any document uploaded will not be saved
+
+  Scenario: AC9  LPA Planning Officer deletes a file prior to save and continue
+    Given a file has been uploaded
+    When LPA Planning Officer deletes the file
+    Then the file is removed
+
+  Scenario: AC10  LPA Planning Officer deletes a file after save and continue
+    Given a file has been uploaded and confirmed
+    And Upload the Planning Officer’s report question is requested
+    When LPA Planning Officer deletes the file
+    Then the file is removed
+
+  Scenario: AC11 LPA Planning Officer returns to the completed Upload the plans used to reach the decision question
+    Given The question 'Upload the Planning Officer’s report' has been completed
+    When the Upload the Planning Officer’s report question is requested
+    Then the information they previously entered is still populated
+
+  Scenario: AC12 Appeal details side panel
+    Given Upload the Planning Officer’s report question is requested
+    Then the appeal details panel is displayed on the right hand side of the page
+
+  @nojs
+  Scenario Outline: AC13 JavaScript Disabled
+    Given Upload the Planning Officer’s report question is requested
+    When valid file '<valid_file>' is successfully uploaded
+    Then progress is made to the task list
+    And Upload the Planning Officer’s report subsection is shown as completed
+    Examples:
+      | valid_file             |
+      | upload-file-valid.tiff |
+
+  Scenario: AC14 Check your answers
+    Given all the mandatory questions for the questionnaire have been completed
+    When Check your Answers is displayed
+    Then Upload the Planning Officer’s report heading and the uploaded file name should be displayed
+
+  Scenario: AC15 Change answers
+    Given a change to answer 'Upload Planning Officers report' is requested from Change your answers page
+    When an answer is saved
+    Then progress is made to the Check Your Answers page
+    And the updated answer is displayed
+
+  # Scenario: AC16 PDF & Horizon
```

Comment:
how do we feel about commented out code in master?

Reply:
Spoke about this with Pippa and chosen this approach for now. Not ideal but as in e2e not as bad in my mind